.github/workflows/pages.yml

```diff
@@ -4,7 +4,7 @@ name: Deploy static content to Pages
 on:
   # Runs on pushes targeting the default branch
   push:
-    branches: ["master"]
+    branches: ["workflow"]
```

Comment:
You should keep this as master, this isn't the same thing as changing the actions branch in repo settings

Reply:
Changed to master